### ISSUE TITLE
LMG_Minigun in multiple pbo's

### ISSUE THREAD

**Issue by GuzzenVonLidl** [acemod/ACE3#2897]
I was just looking through some of the code and noticed that both ace_aircraft & ace_vehicles have the same class with different values, which one is true and is it something you guys are aware of?
if you guys are aware of this just close this ticket

Ace_Aircraft
https://github.com/acemod/ACE3/blob/master/addons/aircraft/CfgWeapons.hpp#L84

`    class LMG_Minigun: LMG_RCWS {`
`        class manual: MGun {`
`            reloadTime = 0.075; //0.015;`
`            dispersion = 0.00093; //0.006;`
`        };`
`        class close: manual {};`
`        class short: close {};`
`        class medium: close {};`
`        class far: close {};`
`    };`

Ace_Vehicles
https://github.com/acemod/ACE3/blob/master/addons/vehicles/CfgWeapons.hpp#L29
`    class LMG_Minigun: LMG_RCWS {`
`        class manual: MGun {`
`            reloadTime = 0.015;`
`            dispersion = 0.006;`
`        };`
`        class close: manual {};`
`        class short: close {};`
`        class medium: close {};`
`        class far: close {};`
`    };`

**Comment by commy2** [comment]
vanilla:
```
	class LMG_Minigun: LMG_RCWS
	{
		class manual: MGun
		{
			reloadTime = 0.033;
			dispersion = 0.0087;
		};
```

**Comment by commy2** [comment]
It has no effect in Air anyway, because there is now a `LMG_Minigun_Heli` which overwrites these changes.
I'd say we just get rid of both.

**Comment by MikeMatrix** [comment]
If the behavior doesn't change by removing them, then they should be removed.

**Comment by commy2** [comment]
fixed in #2955 
Turns out the ACE_Air.pbo took precedence due to loading order.
